### DISCUSSION TITLE
Pressing Esc

### DISCUSSION THREAD

**Discussion by TsKyrk** [WinMerge/winmerge#1960] [General]
I have launched a comparison between two huge folders. The result came after 10mn of processing. While navigating through the results I've pressed on my keyboard's Esc button to close a pop-up message or a context menu or something like that. This resulted in the result window being instantly closed. Now Winmerge is processing these folders again and I though I will use this time to come here an report my misfortune.

Is there any setting available to prevent this behavior ? Other than setting my brain not to press Esc randomly ?

**Comment by sdottaka** [comment]
Please select "Disabled" from the "Close windows with 'Esc'" combo box in the Options window, under the General category, as shown below.

![image](https://github.com/WinMerge/winmerge/assets/98126/188ce77b-c082-41e5-88d2-04228a6bf87a)

**Reply by TsKyrk** [reply]
great ! thank you.
